run.sh

```diff
@@ -346,6 +349,23 @@ check_plugins() {
   done <<< "$(WP plugin list --field=name)"
 }
 
+check_packages() {
+  if [[ ! "${REQUIRE-}" ]]; then
+    h3 "No package dependencies listed"
+    STATUS SKIP
+    return
+  fi
+
+  h3 "Installing composer packages"
+  touch /app/composer.json
```

Comment:
Voluming a composer.json is still possible, but with this feature, you need one less config file, and all the dependencies of your wordpress are listed in a single place.